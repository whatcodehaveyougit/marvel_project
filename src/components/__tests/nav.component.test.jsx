import { describe, expect, test } from "vitest"
import Nav from '../nav/nav.component'
import React from 'react'
import { render } from "@testing-library/react"
import { BrowserRouter as Router } from "react-router-dom";


describe('NavComponentTest', () => {
  test('Does a snapshot of the Nav component', () => {
    const { container } = render(
      <Router>
        <Nav />
      </Router>
    )
    expect(container).toMatchSnapshot()

  })
})