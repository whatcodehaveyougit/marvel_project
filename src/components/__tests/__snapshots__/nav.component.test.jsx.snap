// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavComponentTest > Does a snapshot of the Nav component 1`] = `
<div>
  <div>
    <div>
      <h3
        class="MuiTypography-root MuiTypography-h3 MuiTypography-alignCenter css-47suzj-MuiTypography-root"
      >
        <a
          href="/"
        >
          Marvel Application!
        </a>
      </h3>
    </div>
    <div
      class="navbar-links-container"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
      >
        <a
          href="/"
        >
          Home
        </a>
      </h5>
      <h5
        class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
      >
        <a
          href="/about"
        >
          About
        </a>
      </h5>
    </div>
  </div>
</div>
`;
